projects/clas12/targets/materials.py

```diff
@@ -0,0 +1,226 @@
+	gmaterial.publish(configuration)
+
+	# lHe coolant
+	gmaterial = GMaterial('lHeCoolant')
+	gmaterial.description = 'liquid He coolant for the polarized target cell'
+	# @mariakzurek: Should it be 145 or 147??????
+	gmaterial.density = 0.147  # 0.145 g/cm3  
+	gmaterial.addMaterialWithFractionalMass('G4_He', 1)
+	gmaterial.publish(configuration)
+
+	# NH3
+	NH3_density = 0.867
+	N_mass_fraction=15/18
+	H_mass_fraction=3/18
+	gmaterial = GMaterial('NH3')
+	gmaterial.description = 'NH3 material'
+	gmaterial.density = NH3_density 
+	gmaterial.addMaterialWithFractionalMass('G4_N', N_mass_fraction)
+	gmaterial.addMaterialWithFractionalMass('G4_H', H_mass_fraction)
+	gmaterial.publish(configuration)
+
+	# NH3 target with lHe3 coolant
+	NH3trg_density = 0.6*0.867+0.4*0.145 # 60% of NH3 and 40% of liquid-helium
+	NH3_mass_fraction=0.6*0.867/NH3trg_density 
+	lHe_mass_fraction=0.4*0.145/NH3trg_density
+	gmaterial = GMaterial('NH3target')
+	gmaterial.description = 'solid NH3 target material'
+	gmaterial.density =  NH3trg_density
+	gmaterial.addMaterialWithFractionalMass('NH3', NH3_mass_fraction)
+	gmaterial.addMaterialWithFractionalMass('lHeCoolant', lHe_mass_fraction)
+	gmaterial.publish(configuration)
+
+	# G10 fiberglass
+	gmaterial = GMaterial('G10')
+	gmaterial.description = 'G10 - 1.70 g/cm3'
+	gmaterial.density =  1.70
+	gmaterial.addMaterialWithFractionalMass('G4_Si', 0.283)
+	gmaterial.addMaterialWithFractionalMass('G4_O', 0.323)
+	gmaterial.addMaterialWithFractionalMass('G4_C', 0.364)
+	gmaterial.addMaterialWithFractionalMass('G4_H', 0.030)
+	gmaterial.publish(configuration)
+
+	# lHe gas
+	gmaterial = GMaterial('HeGas')
+	gmaterial.description = 'Upstream He gas'
+	gmaterial.density =  0.000164  # 0.164 kg/m3 
+	gmaterial.addMaterialWithFractionalMass('G4_He', 1)
+	gmaterial.publish(configuration)
+
+	# Al target cell(s)
+	gmaterial = GMaterial('AlCell')
+	gmaterial.description = 'Aluminum for target cells '
+	gmaterial.density =  2.70  # 2.7 g/cm3 
+	gmaterial.addMaterialWithFractionalMass('G4_Al', 1)
+	gmaterial.publish(configuration)
+
+	# Shim coils, need to confirm mass fractions and density
+	NbTi_density=6.63
+	ShimCoil_density=(1.3*8.96+NbTi_density)/2.3
+	Cu_mass_fraction=29/(29+41+22)
+	Nb_mass_fraction=41/(29+41+22)
+	Ti_mass_fraction=22/(29+41+22)
+	gmaterial = GMaterial('ShimCoil')
+	gmaterial.description = 'Cu/NbTi correction coils ratio is 1.3 Cu  to 1 NbTi'
+	gmaterial.density =  ShimCoil_density  #Cu=8.96 g/cm3 NbTi= 4.5 g/cm3 = Nb-47.5 wt % Ti 47.5% of the materials weight is Nb 
+	gmaterial.addMaterialWithFractionalMass('G4_Cu', Cu_mass_fraction)
+	gmaterial.addMaterialWithFractionalMass('G4_Nb', Nb_mass_fraction)
+	gmaterial.addMaterialWithFractionalMass('G4_Ti', Ti_mass_fraction)
+	gmaterial.publish(configuration)
+
+	# Target cup walls with holes, need to confirm mass ratios
+	# @mariakzurek: Do we need it? It is the same as Kel-F. Who is the author of the comment above?
```

Comment:
@maureeungaro Please check this comment

Reply:
Not needed it's a copy. Let's remove it.